### DISCUSSION TITLE
"Issue closed reasons" management

### DISCUSSION THREAD

**Discussion by ov7a** [community/community#52048] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

It would be nice to have an ability to configure "Issue closed reasons" in some way or another to make issue management smoother.

The simplest change is to add an option to select the default reason. Right now it is "Close as completed" but in some projects it would be a lot more reasonable to have "Close as not planned" as a default.

Another feature might be the explicit sorting of reasons (the first being default).

Yet another option might be to have restrictions - for example, only a repository maintainer should have the option to close an issue as completed.

And of course it would be very nice to be able to configure all of this in the `.github` folder (like issue templates).

**Comment by gbakeman** [comment]
I had opened another discussion (#56509) just to find out that it is impossible to currently specify the exact `state_reason` from the GitHub UI. The `state_reason` feature was introduced in the [May 19th 2022 blog post](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/). Hopefully we can use this discussion as a way to organize and track this feature being fully brought to the UI.
